### PR TITLE
Update CONTRIBUTING.md to ensure ports highlight original content

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Large ports must now include details of which parts are original code, to make reviewing easier.

## Why It's Good For The Game

When it comes to ports, the most likely thing to fail is the original code (since the TG code already has had significant testing on TG).

To make reviewing easier, contributors porting code from another codebase should indicate all the parts of their PR that is original code.

## Testing Photographs and Procedure

N/A

## Changelog
:cl:
code: (Github) Large ports must now indicate which part of their port is original code.
/:cl:

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ Any pull-request that does not adequately complete the provided template may be
  - The section labeled "why it's good for the game" should include the reasons behind the changes and how they will be good for the game.
  - The testing section should contain screenshots, videos, and/or reproducible testing procedures showing that the PR works as specified. Pull-requests that ignore this section, or are not tested, may be closed by maintainers. This applies to small PRs that may seem trivial.
  - The changelog should include a short summary of the changes made. If your pull request includes things made by other people, you should list everybody who contributed, including yourself, after the :cl: tag.
+ - Ports with a significant number of file changes (roughly more than 10) must indicate which parts of the code are ported, and which parts are original code.
```
Comment (PowerfulBacon) [inline .github/CONTRIBUTING.md]:
```suggestion
 - Ports should indicate any original code that they have added for the port. This includes highlighting any mass-edit statements done via regex, and any bee specific cnotent that had to be modified to accomodate the changes.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -110,6 +110,7 @@ Any pull-request that does not adequately complete the provided template may be
  - The section labeled "why it's good for the game" should include the reasons behind the changes and how they will be good for the game.
  - The testing section should contain screenshots, videos, and/or reproducible testing procedures showing that the PR works as specified. Pull-requests that ignore this section, or are not tested, may be closed by maintainers. This applies to small PRs that may seem trivial.
  - The changelog should include a short summary of the changes made. If your pull request includes things made by other people, you should list everybody who contributed, including yourself, after the :cl: tag.
+ - Ports should indicate any original code that they have added for the port. This includes highlighting any mass-edit statements done via regex, and any bee specific cnotent that had to be modified to accomodate the changes.
```
Comment (Programs-The-Station) [inline .github/CONTRIBUTING.md]: "Beestation specific c**on**tent"

Comment (PowerfulBacon) [inline .github/CONTRIBUTING.md]:
```suggestion
 - Ports should indicate any original code that they have added for the port. This includes highlighting any mass-edit statements done via regex, and any bee specific content that had to be modified to accomodate the changes.
```

**Comment by Tsar-Salat** [comment]
I think the labeling workflow needs a new check so it applies the Port label so this is easier to search by and apply.

Otherwise that label is unused.

**Comment by PowerfulBacon** [comment]
I didn't.even know we had a port label

**Comment by PowerfulBacon** [comment]
I've already announced this

**Pull request merged**